packages/cascara/src/modules/DataEmail/DataEmail.js

```diff
@@ -28,13 +29,14 @@ const DataEmail = ({
   ...rest
 }) => {
   const { isEditing, formMethods } = useContext(ModuleContext);
+  const { setAriaLabel, setHtmlFor } = getHTMLLabels(isLabeled, label);
```

Comment:
@brianespinosa I'm not sure about this name. Could you advise here?

Reply:
I think `getAccessibleLabelSetters`